docs/content/guides/dagster-pipes/integrating-databricks-with-dagster-pipes.mdx

```diff
@@ -1,6 +1,305 @@
+  - **The `dagster-databricks` library**:
+
+    ```shell
+    pip install dagster-databricks
+    ```
+
+- **In Databricks:**
+  - **An existing Databricks workspace** with an existing project that’s deployed with a Databricks job. If you don’t have this, follow the [Databricks quickstart](https://docs.databricks.com/workflows/jobs/jobs-quickstart.html) to set one up.
+  - **The following information about your Databricks workspace**:
+    - `host` - The host address, starting with `https://`, of your Databricks workspace
+    - `token` - A personal access token for the Databricks workspace Refer to the Databricks API authentication documentation for more info about retrieving these values.
+  - **To have the `dagster-pipes` library available in the Databricks environment.**
+
+---
+
+## Step 1: Create a Databricks job in Dagster
+
+In this step, you’ll create a Dagster asset that, when materialized, opens a Dagster pipes session and launches a Databricks job that executes some external code.
+
+### Step 1.1: Define a Dagster asset
+
+1. Create a new file named `dagster_databricks_pipes.py` in your Dagster project.
+
+2. Copy and paste the following imports into the file:
+
+   ```python
+   # dagster_databricks_pipes.py
+
+   from dagster import AssetExecutionContext, Definitions, EnvVar, asset
+   from dagster_databricks import PipesDatabricksClient
+   from databricks.sdk.service import jobs
+   ```
+
+3. Next, you’ll define the asset. Copy and paste the following below the imports in `dagster_databricks_pipes.py`:
+
+   ```python
+   # dagster_databricks_pipes.py
+
+   @asset
+   def databricks_asset(context: AssetExecutionContext, pipes_databricks: PipesDatabricksClient):
+       task = jobs.SubmitTask.from_dict({
+           "new_cluster": { ... },
+           "libraries": [
+               # must include dagster-pipes
+               {"pypi": {"package": "dagster-pipes"}},
+           ],
+           "task_key": "some-key",
+           "spark_python_task": {
+               "python_file": "dbfs:/existing_databricks_code.py",   # location of target code file
+               "source": jobs.Source.WORKSPACE,
+           }
+       })
+   ```
+
+   Here’s what we did in this example:
+
+   - Created an asset named `databricks_asset`
+   - Provided <PyObject object="AssetExecutionContext" /> as the `context` argument to the asset. This object provides system information such as resources, config, and logging. We’ll come back to this a bit later in this section.
+   - Specified a resource for the asset to use, <PyObject module="dagster_databricks" object="PipesDatabricksClient" />. We’ll also come back to this in a little bit.
+   - Defined a Databricks `SubmitTask` specification in the asset body. When the asset is materialized, this specification will be passed to the Databricks resource (<PyObject module="dagster_databricks" object="PipesDatabricksClient" />), which will use it to launch a Databricks job.
```

Comment:
"Defined a Databricks `SubmitTask` object in the asset body using the official Databricks Python SDK. The `PipesDatabricksClient` accepts this object and injects environnment variables and a cluster logging specification into it before submission to the Databricks API to launch a job."